spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/SpringDocAutoConfiguration.java

```diff
@@ -58,30 +61,29 @@ public SpringDocConfiguration(SpringDocConfigProperties springDocConfigPropertie
      * @return a web security customizer with security settings for SpringDoc
      */
     @Bean
+    @Qualifier("springDocWebSecurityCustomizer")
     public WebSecurityCustomizer springDocCustomizer() {
-        return (webSecurity -> {
-            String apiDocsPathContext = getApiDocsPathContext();
-            webSecurity.ignoring().antMatchers(
-                    SWAGGER_UI_CONTEXT,
-                    apiDocsPathContext + "/**",
-                    swaggerUiConfigProperties.getPath(),
-                    swaggerUiConfigProperties.getConfigUrl(),
-                    swaggerUiConfigProperties.getValidatorUrl(),
-                    swaggerUiConfigProperties.getOauth2RedirectUrl(),
-                    springDocConfigProperties.getWebjars().getPrefix(),
-                    springDocConfigProperties.getWebjars().getPrefix() + "/**");
-        });
+        return (webSecurity -> webSecurity.ignoring().antMatchers(
+                SWAGGER_UI_CONTEXT,
+                getApiDocsPathContext() + "/**",
+                swaggerUiConfigProperties.getPath(),
+                swaggerUiConfigProperties.getConfigUrl(),
+                swaggerUiConfigProperties.getValidatorUrl(),
+                swaggerUiConfigProperties.getOauth2RedirectUrl(),
+                springDocConfigProperties.getWebjars().getPrefix(),
+                springDocConfigProperties.getWebjars().getPrefix() + "/**"));
     }
 
     /**
-     * Creates a filter registration bean which decodes the JSON of ApiDocs and SwaggerUi so that the SpringDoc frontend is able
+     * Applies {@link SpringDocJsonDecodeFilter} to the filter chain which decodes the JSON of ApiDocs and SwaggerUi so that the SpringDoc frontend is able
      * to read it. Spring Cloud Data Flow however requires the JSON to be escaped and wrapped into quotes, because the
      * Angular Ui frontend is using it that way.
      *
      * @return a filter registration bean which unescapes the content of the JSON endpoints of SpringDoc before it is returned.
      */
     @Bean
-    public FilterRegistrationBean<SpringDocJsonDecodeFilter> jsonDecodeFilterRegistration() {
+    @Qualifier("springDocJsonDecodeFilterRegistration")
```

Comment:
We can remove `@Qualifier` here as the name is the same so this is redundant.

Reply:
Okay - I just removed it. 👍